### ISSUE TITLE
Old documentation for SSL/TLS

### ISSUE THREAD

**Issue by kmcgivney** [franzinc/aserve#12]
In the allegroserve documentation on SSL/TLS, https://github.com/franzinc/aserve/blob/master/doc/aserve.md#ssltls, there's some very old information about TLS (old with respect to TLS and with respect to aserve's support for TLS). In that section, it links to https://franz.com/support/documentation/current/doc/operators/socket/make-ssl-server-stream.htm, however *that* is also out of date, claiming it only goes up to :tlsv1. I believe that the best documentation you have on this is over here: https://franz.com/support/documentation/10.1/doc/operators/socket/make-ssl-client-context.htm which includes keywords up through :tlsv1.2.

I believe we should remove the outdated discussion of TLS and either explain the argument options here, or link to a page that does that. Thanks.

**Comment by franzinc** [comment]
Thanks.  We will look into it.

**Comment by kmcgivney** [comment]
Thanks @jkftiger348 . I was already using that on the down-low, to specify things like **ciphers** and **certificate**. But it's good that it's documented now. This issue is still open because the file aserve.md#ssltls is out of date with regards to supported SSL/TLS versions. I'm sure there's a way to reword it to account for this change.
